sirepo_bluesky/sirepo_bluesky.py

```diff
@@ -70,7 +71,8 @@ def auth(self, sim_type, sim_id):
 
     def copy_sim(self, sim_name):
         """ Create a copy of the current simulation. Returns a new instance of SirepoBluesky. """
-        assert self.sim_id
+        if not self.sim_id:
+            raise NotImplementedError
```

Comment:
I think a `Value` or `RuntimeError` will be more appropriate here. Please also make sure you add a message with a summary of the problem (in this case it will be useful to point that `self.sim_id` is False/None/'' and probably the method name).